packages/native/types/base.d.ts

```diff
@@ -32,7 +28,7 @@ export type Interpolation<MergedProps = undefined> =
   | ArrayInterpolation<MergedProps>
   | FunctionInterpolation<MergedProps>
 
-type ReactNativeTags =
+export type ReactNativeComponentNames =
```

Comment:
I was originally going to use `ReactNativeStylableComponentNames` but that seemed overly verbose.

Reply:
@patsissons instead of hardcoding the names like this, what do you think of:

```typescript
type ReactNativeComponents = Pick<ReactNative, {
      [Key in keyof ReactNative]: ReactNative[Key] extends ComponentType ? Key : never
  }[keyof ReactNative]>
type ReactNativeComponentNames = keyof ReactNativeComponents
```

I'm using it like this:
```typescript
  type ActivityIndicatorProps = ComponentPropsWithoutRef<ReactNativeComponents["ActivityIndicator"]>
```

And it says the type is `type ActivityIndicatorProps = RN.ActivityIndicatorProps`, which seems about right.

Reply:
the hardcoded names are specifically to represent the [hardcoded component exports](https://github.com/emotion-js/emotion/blob/master/packages/native/src/index.js#L8-L57) in the native module. It's possible that the two sets are the same, but i wanted the styles to accurately match the module, at least for this first release.